src/clj/harja/kyselyt/urakat.sql

```diff
@@ -135,6 +135,11 @@ UPDATE urakka
 SET lyhyt_nimi = :lyhytnimi
 WHERE id = :urakka;
 
+-- name: hae-mh-urakoiden-urakka-numerot
+SELECT u.urakkanro, u.lyhyt_nimi
+FROM urakka u
+WHERE u.tyyppi = 'teiden-hoito' AND u.velho_oid IS NULL;
```

Comment:
WHERE-ehtoon voisi lisätä `AND urakkanro IS NOT NULL`. Tuotannon kannassa on testiurakka, jonka urakkanro on null. (tarkoituksella, sillä myös tunnistetaan että se on testiurakka eikä sen dataa käytetä missään raportissa tms.)

Reply:
ok